src/components/Root/index.tsx

```diff
@@ -20,7 +21,13 @@ const Root = (): React.ReactElement => (
   <ThemeProvider theme={styledTheme}>
     <Provider store={store}>
       <MuiThemeProvider theme={theme}>
-        <ConnectedRouter history={history}>{<App>{wrapInSuspense(<AppRoutes />, <Loader />)}</App>}</ConnectedRouter>
+        <ConnectedRouter history={history}>
+          {
+            <Sentry.ErrorBoundary fallback={GlobalErrorBoundary}>
+              <App>{wrapInSuspense(<AppRoutes />, <Loader />)}</App>
+            </Sentry.ErrorBoundary>
+          }
```

Comment:
I know this is in progress, but why the curly brackets?

Reply:
fixed